dartagnan/src/main/java/com/dat3m/dartagnan/program/event/Tag.java

```diff
@@ -327,4 +324,74 @@ public static String storeMO(String mo) {
             }
         }
     }
+
+    // =============================================================================================
+    // ========================================= Vulkan ============================================
+    // =============================================================================================
+    public static final class Vulkan {
+        // Scopes
+        public static final String SUB_GROUP = "SG";
+        public static final String WORK_GROUP = "WG";
+        public static final String QUEUE_FAMILY = "QF";
+        public static final String DEVICE = "DV";
+        public static final String PRIVATE = "PRIV";
+        public static final String NON_PRIVATE = "NONPRIV";
+        // Memory orders
+        public static final String ATOM = "ATOM";
+        public static final String ACQUIRE = "ACQ";
+        public static final String RELEASE = "REL";
+        public static final String ACQ_REL = "ACQ_REL";
```

Comment:
`ACQ_REL` [does not exists in the alloy model](https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/66c31d9cfcc03b1f5231779fbf3c42dc983b62e2/alloy/spirv.als#L60). For those instructions that you are assigning `ACQ_REL` while parsing, you should add both `ACQ` and `REL`. This is causing problems like the following
```
test9-filter.litmus

[0] thread void 0()
0:       ThreadStart
1:       store(*a, 1int, AV)
2:       cbar.acq_rel.wg.semsc00 := fence_id[0int] // [WG, ACQ_REL, F, SEMSC0, cbar.acq_rel.wg.semsc00, __NOOPT, _]
3:    END_OF_T0:

[1] thread void 1()
4:       ThreadStart
5:       cbar.acq_rel.wg.semsc00 := fence_id[0int] // [WG, ACQ_REL, F, SEMSC0, cbar.acq_rel.wg.semsc00, __NOOPT, _]
6:       store(*b, 1int, REL)
7:    END_OF_T1:

[2] thread void 2()
8:       ThreadStart
9:       r0 <- 0int
10:      r1 <- 0int
11:      r0 = load(*b, ACQ)
12:      r1 = load(*a, VIS)
13:   END_OF_T2:
```
Here, `membar` is supposed to math the WMM rule
```
    ([REL & F]; po?; [CBAR]; (syncbar & sr \ id); [CBAR]; po?; [ACQ & F])                               // fence->cbar->cbar->fence,
```
both at the beginning and the end, but it does not because it has neither `ACQ` nor `REL`

Reply:
After adding `ACQ` and `REL` to `ACQ_REL` events, the test refered above is still not pass, there should be some other issues I need to check.